src/main/java/taskItems/Course.java

```diff
@@ -0,0 +1,15 @@
+package taskItems;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
```

Comment:
`@Getter`, `@Setter` and `@ToString` could be replaced with `@Data`

Reply:
done